apps/image_augmentation/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ find_package(OpenCV QUIET)
 find_package(AMDRPP QUIET)
 
 # Application Includes & Libraries
-include_directories(${ROCM_PATH}/${CMAKE_INSTALL_INCLUDEDIR}/mivisionx)
+include_directories(${ROCM_PATH}/${CMAKE_INSTALL_INCLUDEDIR}/mivisionx ${ROCM_PATH}/${CMAKE_INSTALL_INCLUDEDIR}/mivisionx/rocal)
```

Comment:
@rrawther Seems like there is a typo. ```${ROCM_PATH}/${CMAKE_INSTALL_INCLUDEDIR}/mivisionx``` is typed twice

Reply:
The second one is for rocal include and is needed